contracts/actions/curve/lp/CurveDeposit.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../helpers/CurveHelper.sol";
+import "../../../utils/TokenUtils.sol";
+import "../../../utils/SafeMath.sol";
+import "../../ActionBase.sol";
+
+contract CurveDeposit is ActionBase, CurveHelper {
+    using TokenUtils for address;
+    using SafeMath for uint256;
+
+    struct Params {
+        address sender;         // address where to pull tokens from
+        address receiver;       // address that will receive the LP tokens
+        address depositTarget;  // pool contract or zap deposit contract in which to deposit
+        address lpToken;        // LP token of the pool
+        bytes4 sig;             // target contract function signature
+        uint256 minMintAmount;  // minimum amount of LP tokens to accept
+        uint256[] amounts;      // amount of each token to deposit
+        address[] tokens;       // tokens to deposit, needed for token approval
+        bool useUnderlying;     // some contracts take this aditional parameter
+    }
+
+    function executeAction(
+        bytes[] memory _callData,
+        bytes[] memory _subData,
+        uint8[] memory _paramMapping,
+        bytes32[] memory _returnValues
+    ) public payable virtual override returns (bytes32) {
+        Params memory params = parseInputs(_callData);
+        params.sender = _parseParamAddr(params.sender, _paramMapping[0], _subData, _returnValues);
+        params.receiver = _parseParamAddr(params.receiver, _paramMapping[1], _subData, _returnValues);
+        params.depositTarget = _parseParamAddr(params.depositTarget, _paramMapping[2], _subData, _returnValues);
+        params.lpToken = _parseParamAddr(params.lpToken, _paramMapping[3], _subData, _returnValues);
+        params.minMintAmount = _parseParamUint(params.minMintAmount, _paramMapping[4], _subData, _returnValues);
+        
+        uint256 N_COINS = params.amounts.length;
```

Comment:
Only constant name with capital letters and underscore

Reply:
dOne